ui/src/shared/components/DygraphLegend.tsx

```diff
@@ -10,7 +10,7 @@ import * as actions from 'src/dashboards/actions'
 import {SeriesLegendData} from 'src/types/dygraphs'
 import DygraphLegendSort from 'src/shared/components/DygraphLegendSort'
 
-import {makeLegendStyles, removeMeasurement} from 'src/shared/graphs/helpers'
```

Comment:
Should delete the code for remove measurement also.

Reply:
I actually did initially delete the `removeMeasurement` code but turns out some other components are using it, so I left it alone